packages/agent/src/agent/http/transforms.ts

```diff
@@ -9,37 +9,38 @@ import {
   Nonce,
 } from './types';
 
-const NANOSECONDS_PER_MILLISECONDS = BigInt(1_000_000);
+const NANOSECONDS_PER_MILLISECOND = BigInt(1_000_000);
+const NANOSECONDS_PER_SECOND = NANOSECONDS_PER_MILLISECOND * BigInt(1_000);
+const SECONDS_PER_MINUTE = BigInt(60);
 
 const REPLICA_PERMITTED_DRIFT_MILLISECONDS = 60 * 1000;
 
 export class Expiry {
-  private readonly _value: bigint;
+  constructor(private readonly _value: bigint) {}
```

Comment:
```suggestion
  private constructor(private readonly _value: bigint) {}
```

Reply:
Might as well make this private if we're shifting to the more explicit constructor